Test-Source/Invoke-DockerPush.Tests.ps1

```diff
@@ -1,28 +1,26 @@
 Import-Module -Force $PSScriptRoot/../Source/Docker.Build.psm1
+Import-Module -Global -Force $PSScriptRoot/Docker.Build.Tests.psm1
 Import-Module -Global -Force $PSScriptRoot/MockReg.psm1
+
 . "$PSScriptRoot\..\Source\Private\CommandResult.ps1"
 
 Describe 'docker push' {
 
     Context 'Push an image' {
 
-        BeforeAll {
-            $script:moduleName = (Get-Item $PSScriptRoot\..\Source\*.psd1)[0].BaseName
-        }
-
         BeforeEach {
             Initialize-MockReg
-            $code = {
+            $returnExitCodeZero = {
```

Comment:
There are several occurrences of this piece of code with similar logic. Perhaps include it in at global sector?

Reply:
I though about it too, I haven't come up with a good way to do it :/

Reply:
We'll refactor this part some time later.